httpserver-rs/Cargo.toml

```diff
@@ -1,5 +1,8 @@
 [package]
 name = "wasmcloud-provider-httpserver"
+description = "implementation of http server for wasmcloud, using warp"
```

Comment:
Fine w/ this for now, if we want to keep publishing it might be good to keep a common description format with the contract ID